week-3/implement/get-angle-type.js

```diff
@@ -21,3 +21,27 @@
 // Identify Reflex Angles:
 // When the angle is greater than 180 degrees and less than 360 degrees,
 // Then the function should return "Reflex angle"
+
+function triangleAngle(degrees){
```

Comment:
good job, but you didnt take care of 3 scenario.

- a negative number
- 360
- and a number greater than 360

Reply:
changed the function according to those scenarios